files/jsons/Policies/Policy_Table_Update/ptu_file_with_full_app_id_supported.json

```
@@ -974,6 +974,15 @@
                 "NORMAL": 5,
                 "VOICECOM": 5
             },
+            "subtle_notifications_per_minute_by_priority": {
+                "EMERGENCY": 60,
+                "NAVIGATION": 20,
+                "PROJECTION": 20,
+                "VOICECOM": 30,
+                "COMMUNICATION": 15,
+                "NORMAL": 10,
+                "NONE": 0
+            },

```

```suggestion
            "notifications_per_minute_by_priority": {
                "COMMUNICATION": 5,
                "EMERGENCY": 5,
                "NAVIGATION": 5,
                "NONE": 5,
                "NORMAL": 5,
                "VOICECOM": 5
            },
```
minor suggestion for consistency with the containing file